api/search.py

```
@@ -343,6 +344,82 @@ def search():
+            for cp in cli_projects:
+                ws_name = cp["workspace_name"] or cp["project_id"][:12]
+                for session in cp["sessions"]:
+                    meta = session.get("meta", {})
+                    session_id = session["session_id"]
+                    created_ms: int = meta.get("createdAt") or int(datetime.now().timestamp() * 1000)
+                    session_name = meta.get("name") or f"Session {session_id[:8]}"
+
+                    try:
+                        messages = traverse_blobs(session["db_path"])
+                    except Exception:
+                        continue
```

Ruff (0.15.5)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)